main/libmemcached/APKBUILD

```diff
@@ -1,49 +1,45 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
-pkgname=libmemcached
-pkgver=1.0.18
-pkgrel=1
+pkgname="libmemcached"
+pkgver="1.0.18"
+pkgrel=2
 pkgdesc="Client library and command line tools for memcached server"
 url="http://libmemcached.org/"
 arch="all"
 license="BSD"
 depends=""
-depends_dev=""
-makedepends="$depends_dev cyrus-sasl-dev"
+depends_dev="cyrus-sasl-dev"
+makedepends="${depends_dev} memcached-dev"
 install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs"
-source="https://launchpad.net/libmemcached/${pkgver%.*}/${pkgver}/+download/libmemcached-${pkgver}.tar.gz
-	musl-fixes.patch
-	"
+subpackages="${pkgname}-dev ${pkgname}-doc ${pkgname}-libs"
+source="https://launchpad.net/${pkgname}/${pkgver%.*}/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz
+	musl-fixes.patch"
 
-_builddir="$srcdir"/libmemcached-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+builddir="${srcdir}/${pkgname}-${pkgver}"
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
 	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
+		--build="$CBUILD" \
+		--host="$CHOST" \
 		--prefix=/usr \
 		--disable-static \
 		--enable-sasl \
 		--enable-libmemcachedprotocol \
-		--enable-memaslap \
-		|| return 1
-	make || return 1
+		--enable-memaslap
+	make
 }
 
-package() {
-	cd "$_builddir"
-	make DESTDIR="$pkgdir" install || return 1
+check() {
+	# Tests are broken as of 1.0.18
 	return 0
```

Comment:
If tests are broken, then disable check adding `options="!check"  # tests are broken` at top-level.

Reply:
Is there some difference that's important about how `check` is disabled?

Reply:
Of course there is, that’s why we have `options` – we can easily analyse what abuilds don’t have check enabled or implemented.